airbyte-integrations/connectors/source-sonar-cloud/source_sonar_cloud/spec.yaml

```diff
@@ -0,0 +1,48 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/sonar-cloud
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Sonar Cloud Spec
+  type: object
+  required:
+    - user_token
+    - organization
+    - component_keys
+  additionalProperties: true
+  properties:
+    user_token:
+      title: User Token
+      type: string
+      description: >-
+        Your User Token. See <a
+        href="https://docs.sonarcloud.io/advanced-setup/user-accounts/">here</a>. The token is
+        case sensitive.
+      airbyte_secret: true
+    organization:
+      title: Organization
+      type: string
+      description: >-
+        Organization key. See <a
+        href="https://docs.sonarcloud.io/appendices/project-information/#project-and-organization-keys">here</a>.
+      examples:
+        - airbyte
+    component_keys:
+      title:  Component Keys
+      type: string
+      description: Comma-separated list of component keys.
+      examples:
+        - airbyte-ws-order,airbyte-ws-checkout
```

Comment:
Change this to type array

Reply:
Switching to array broke the connector, the api waiting for something like this: componentKeys=component-one,component2
However, the array return this: componentKeys=%5B'component-one',%20'component2'%

I will keep string for now.

Reply:
Fixed.